test/functional/rpc_fundrawtransaction.py

```diff
@@ -980,7 +980,7 @@ def test_transaction_too_large(self):
             outputs[recipient.getnewaddress()] = 0.1
         wallet.sendmany("", outputs)
         self.generate(self.nodes[0], 10)
-        assert_raises_rpc_error(-4, "Transaction too large", recipient.fundrawtransaction, rawtx)
+        assert_raises_rpc_error(-4, "Insufficient funds", recipient.fundrawtransaction, rawtx)
```

Comment:
`Insufficient funds` might be a confusing error message since the wallet would also report having sufficient funds at the same time. Perhaps we could return a more specific error, if the last attempt returns from `eligible_results` being empty e.g. `Cannot fund transaction, wallet too fragmented`.